tests/test_preprocess.py

```diff
@@ -0,0 +1,79 @@
+"""Test the neurots.preprocess functions."""
+import json
+from pathlib import Path
+
+import pytest
+
+from neurots import preprocess
+from neurots.utils import convert_from_legacy_neurite_type
+
+DATA_PATH = Path(__file__).parent / "data"
+
+
+@pytest.mark.parametrize(
```

Comment:
I understand that this is convenient, but boy is it hard to debug when sage hits the fan, when you have all these combinations.